.circleci/base-install.sh

```diff
@@ -51,7 +51,13 @@ else
 
 
     # If we skip the yarn install, postinstall may still be needed on any workspace that have changed
-    # since the base docker image was built.
+    # since the base docker image was built. We exclude wome workspaces because their post install is just
+    # another build call, which will be taken care of in a following step.
     set -x
-    yarn workspaces foreach --since=$(cat base_ref) -R run postinstall
+    yarn workspaces foreach \
```

Comment:
This was more reliable. The includes.list uses the output of `modules-to-test.js`, which as it turns out is a bit more robust than just using --since.